Gopkg.toml

```diff
@@ -38,13 +38,9 @@ required = [
   source = "github.com/libopenstorage/external-storage"
   version = "=5.1.0-openstorage"
 
-[[constraint]]
+[[override]]
```

Comment:
Will remove this, once I fix the the deps in torpedo

Reply:
can this be changed now?